### PR TITLE
Remove unused `SerialisedTxParts`

### PR DESCRIPTION
**Pull request by Anviking**
- [x] Remove unused `SeralisedTxParts`

### Issue Number

None.

### CODE REVIEW

File: lib/wallet/src/Cardano/Wallet/Primitive/Types/Tx/SealedTx.hs [hunk 1]
```diff
@@ -295,27 +292,12 @@ unPersistMock bs
...
-getSerialisedTxParts (SealedTx _ (InAnyCardanoEra _ tx) _) = SerialisedTxParts
-    { serialisedTxBody = serialiseToCBOR $ Cardano.getTxBody tx
-    , serialisedTxWitnesses = serialiseToCBOR <$> Cardano.getTxWitnesses tx
-    }
-
 -- | A serialised transaction that may be only partially signed, or even
 -- invalid.
 newtype SerialisedTx = SerialisedTx { payload :: ByteString }
     deriving stock (Show, Eq, Generic, Ord)
     deriving newtype (Semigroup, Monoid, ByteArray, ByteArrayAccess, NFData)
```
Comment (Anviking) [inline lib/wallet/src/Cardano/Wallet/Primitive/Types/Tx/SealedTx.hs:299]: Btw, `tx1 <> tx2` 🙈

File: lib/wallet/src/Cardano/Wallet/Primitive/Types/Tx/SealedTx.hs [hunk 2]
```diff
@@ -295,27 +292,12 @@ unPersistMock bs
   where
     (header, body) = B8.splitAt (B8.length mockSealedTxMagic) bs
 
--- | Get the serialised transaction body and witnesses from a 'SealedTx'.
-getSerialisedTxParts :: SealedTx -> SerialisedTxParts
-getSerialisedTxParts (SealedTx _ (InAnyCardanoEra _ tx) _) = SerialisedTxParts
-    { serialisedTxBody = serialiseToCBOR $ Cardano.getTxBody tx
-    , serialisedTxWitnesses = serialiseToCBOR <$> Cardano.getTxWitnesses tx
-    }
-
```
Comment (Unisay) [inline lib/wallet/src/Cardano/Wallet/Primitive/Types/Tx/SealedTx.hs:304]: Do I understand it right that this function isn't used anywhere?

**Review by Unisay — APPROVED** [review]
Nice cleanup!

**Review by jonathanknowles — APPROVED** [review]

**Review by paweljakubas — APPROVED** [review]
:+1:

File: lib/wallet/src/Cardano/Wallet/Primitive/Types/Tx/SealedTx.hs (hunk 2 above)
Comment (Anviking) [reply lib/wallet/src/Cardano/Wallet/Primitive/Types/Tx/SealedTx.hs:304]: Indeed

**Pull request merged**